packages/@apphosting/adapter-nextjs/src/utils.ts

```diff
@@ -133,44 +130,32 @@ export function createMetadata(nextVersion: string): Metadata {
 
 // generate bundle.yaml
 async function generateBundleYaml(
-  outputBundleOptions: OutputBundleOptions,
-  nextBuildDirectory: string,
+  opts: OutputBundleOptions,
   cwd: string,
   nextVersion: string,
 ): Promise<void> {
-  const manifest = await readRoutesManifest(nextBuildDirectory);
-  const headers = manifest.headers.map((it) => ({ ...it, regex: undefined }));
-  const redirects = manifest.redirects
-    .filter((it) => !it.internal)
-    .map((it) => ({ ...it, regex: undefined }));
-  const beforeFileRewrites = Array.isArray(manifest.rewrites)
-    ? manifest.rewrites
-    : manifest.rewrites?.beforeFiles || [];
-  const rewrites = beforeFileRewrites.map((it) => ({ ...it, regex: undefined }));
-  await writeFile(
-    outputBundleOptions.bundleYamlPath,
-    yamlStringify({
-      headers,
-      redirects,
-      rewrites,
-      runCommand: `node ${normalize(relative(cwd, outputBundleOptions.serverFilePath))}`,
-      neededDirs: [normalize(relative(cwd, outputBundleOptions.outputDirectoryBasePath))],
-      staticAssets: [normalize(relative(cwd, outputBundleOptions.outputPublicDirectoryPath))],
-      env: [],
-      metadata: createMetadata(nextVersion),
-    }),
-  );
+  await mkdir(opts.outputDirectoryBasePath);
+  const outputBundle: OutputBundleConfig = {
+    version: "v1",
+    serverConfig: {
```

Comment:
How will the env var section be taken cared of here?

Reply:
It will be done the same way things are done in the angular adapter, we'll only add it if needed. For now we don't have any env var needs and env var is optional so there is no need to add a field for it yet